components/server/src/auth/auth-provider-service.ts

```diff
@@ -244,7 +244,11 @@ export class AuthProviderService {
         return this.config.hostUrl.with({ pathname }).toString();
     };
 
-    async isHostReachable(host: string) {
-        return await isReachable(host, { timeout: 2000 });
+    async isHostReachable(host: string): Promise<boolean> {
+        try {
+            const resp = await fetch(`https://${host}`, { timeout: 2000, method: "HEAD" });
+            return resp.ok;
+        } catch (error) {}
```

Comment:
Let's log this out. I think it can help us in debugging some aspects of dedciated

Reply:
Separate PR please. I need the yarn.lock changes to continue :pray: